example/src/androidTest/java/ably/io/tutorial/ApplicationTest.java

```diff
@@ -0,0 +1,13 @@
+package ably.io.tutorial;
+
+import android.app.Application;
+import android.test.ApplicationTestCase;
+
+/**
+ * <a href="http://d.android.com/tools/testing/testing_android.html">Testing Fundamentals</a>
+ */
+public class ApplicationTest extends ApplicationTestCase<Application> {
+    public ApplicationTest() {
+        super(Application.class);
+    }
+}
```

Comment:
Please make sure there are trailing newlines on all files that are not auto-generated